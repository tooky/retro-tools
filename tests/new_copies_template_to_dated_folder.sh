EDITOR=fake-editor retro_date=1997-06-05 retro new
ls doc/retro/*/*/*
cat doc/retro/1997/06/05.md
