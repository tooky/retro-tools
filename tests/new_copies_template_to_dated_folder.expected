EDITOR=fake-editor retro_date=1997-06-05 retro new
EDITOR doc/retro/1997/06/05.md
doc/retro/1997/06/05.md
ls doc/retro/*/*/*
doc/retro/1997/06/05.md
cat doc/retro/1997/06/05.md
# 1997-06-05
## Who

## What happened

## Insights

## Actions
